<html><h3>:x: ci:test - 2 out of 6 jobs passed in 1 hour 3 minutes 17 seconds 21 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/7aa4bfd3c78f654417186fb3cf88f2b0f347f6e9">7aa4bfd3c78f654417186fb3cf88f2b0f347f6e9</a></p>2 out of 6 jobs PASSED<div><h4>4 Failed Jobs:</h4><ul><li><a href="https://test-1-14.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/71/"><strike><strong>test-subrepository-acceptance-pullrequest(7.1.x)</strong></strike></a></li><li><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/599/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/functional-smoke-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/598/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-assemble-jdk8</strong></strike></a></li><li><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/600/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-integration-mysql57-jdk8</strong></strike></a></li></ul></div><details><summary><strong>2 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-19.liferay.com/job/test-subrepository-acceptance-pullrequest-dist(7.1.x)/62/">test-subrepository-acceptance-pullrequest-dist(7.1.x)/jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/84/">test-subrepository-acceptance-pullrequest-validation(7.1.x)/jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-14.liferay.com/userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/71/jenkins-report.html">here</a>.</h5><hr/><h4>Failures (upstream comparison is not available):</h4><ol><li><div><h5><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/599/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/functional-smoke-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>3 Tests Failed.</p></div><ol><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/599//consoleText">AXIS_VARIABLE=0,label_exp=!master #599</a><ol><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/599//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">PortalLogAssertorTest.testScanXMLLog</a><pre><code>junit.framework.AssertionFailedError: 
FrameworkEvent ERROR 
graphql.AssertException: Duplicated definition for field 'postAccountGroupAccountGroupCriterion' in type 'Mutation'
	at graphql.schema.GraphQLObjectType.buildDefinitionMap(GraphQLObjectType.java:72)
	at graphql.schema.GraphQLObjectType.&lt;init&gt;(GraphQLObjectType.java:59)
	at graphql.schema.GraphQLObjectType$Builder.build(GraphQLObjectType.java:233)
	at graphql.annotations.processor.retrievers.GraphQLTypeRetriever.getGraphQLType(GraphQLTypeRetriever.java:85)
	at graphql.annotations.processor.retrievers.GraphQLObjectHandler.getObject(GraphQLObjectHandler.java:33)
	at com.liferay.portal.vulcan.internal.graphql.servlet.GraphQLServletExtender$ServletDataServiceTrackerCustomizer.addingService(GraphQLServletExtender.java:154)
	at com.liferay.portal.vulcan.internal.graphql.servlet.GraphQLServletExtender$ServletDataServiceTrackerCustomizer.addingService(GraphQLServletExtender.java:133)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:943)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:1)
	at org.osgi.util.tracker.AbstractTracked.trackAdding(AbstractTracked.java:256)
	at org.osgi.util.tracker.AbstractTracked.track(AbstractTracked.java:229)
	at org.osgi.util.tracker.ServiceTracker$Tracked.serviceChanged(ServiceTracker.java:903)
	at org.eclipse.osgi.internal.serviceregistry.FilteredServiceListener.serviceChanged(FilteredServiceListener.java:109)
	at or...</code></pre></div></li><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/599//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanOSGiLog">PortalLogAssertorTest.testScanOSGiLog</a><pre><code>junit.framework.AssertionFailedError: 
Portal log assert failure, OSGi log found: /opt/dev/projects/github/liferay-portal-7.1.x/bundles/osgi/state/1560285369921.log:
!SESSION 2019-06-11 20:38:12.145 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_121
java.vendor=Oracle Corporation
BootLoader constants: OS=linux, ARCH=x86_64, WS=gtk, NL=en_US

!ENTRY com.liferay.portal.vulcan.impl 4 0 2019-06-11 20:38:12.145
!MESSAGE FrameworkEvent ERROR
!STACK 0
graphql.AssertException: Duplicated definition for field 'postAccountGroupAccountGroupCriterion' in type 'Mutation'
	at graphql.schema.GraphQLObjectType.buildDefinitionMap(GraphQLObjectType.java:72)
	at graphql.schema.GraphQLObjectType.&lt;init&gt;(GraphQLObjectType.java:59)
	at graphql.schema.GraphQLObjectType$Builder.build(GraphQLObjectType.java:233)
	at graphql.annotations.processor.retrievers.GraphQLTypeRetriever.getGraphQLType(GraphQLTypeRetriever.java:85)
	at graphql.annotations.processor.retrievers.GraphQLObjectHandler.getObject(GraphQLObjectHandler.java:33)
	at com.liferay.portal.vulcan.internal.graphql.servlet.GraphQLServletExtender$ServletDataServiceTrackerCustomizer.addingService(GraphQLServletExtender.java:154)
	at com.liferay.portal.vulcan.internal.graphql.servlet.GraphQLServletExtender$ServletDataServiceTrackerCustomizer.addingService(GraphQLServletExtender.java:133)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:943)
	at org.osgi.util.tracker...</code></pre></div></li><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/599//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PortalSmoke_Smoke_">LocalFile.PortalSmoke#Smoke</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-14/1560282194369/test-subrepository-acceptance-pullrequest(7.1.x)/71/functional-smoke-tomcat90-mysql57-jdk8/0/0/LocalFile.PortalSmoke_Smoke/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-14/1560282194369/test-subrepository-acceptance-pullrequest(7.1.x)/71/functional-smoke-tomcat90-mysql57-jdk8/0/0/LocalFile.PortalSmoke_Smoke/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-14/1560282194369/test-subrepository-acceptance-pullrequest(7.1.x)/71/functional-smoke-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/598/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-assemble-jdk8</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/598//consoleText">AXIS_VARIABLE=0,label_exp=!master #598</a><div><p>Please fix <strong>semantic versioning</strong> on <strong><a href="https://github.com/luca-pellizzon/com-liferay-commerce/tree/fixJS">luca-pellizzon/fixJS</a></strong><pre><code>     [exec] 			+   annotated  com.liferay.portal.kernel.search.Indexable
     [exec] 				+   property   type='REINDEX'
     [exec] 		+   method     getTagIds(java.lang.String)
     [exec] 			+   access     abstract
     [exec] 			+   annotated  com.liferay.portal.kernel.transaction.Transactional
     [exec] 				+   property   propagation='SUPPORTS'
     [exec] 				+   property   readOnly=true
     [exec] 			+   return     long[]
     [exec] 		+   method     getTagsSize(long,java.lang.String)
     [exec] 			+   access     abstract
     [exec] 			+   annotated  com.liferay.portal.kernel.transaction.Transactional
     [exec] 				+   property   propagation='SUPPORTS'
     [exec] 				+   property   readOnly=true
     [exec] 			+   return     int
     [exec] 		~   method     incrementAssetCount(long,long)
     [exec] 			+   annotated  com.liferay.portal.kernel.search.Indexable
     [exec] 				+   property   type='REINDEX'
     [exec] 		+   method     searchTags(long[],java.lang.String,int,int,com.liferay.portal.kernel.search.Sort)
     [exec] 			+   access     abstract
     [exec] 			+   annotated  com.liferay.portal.kernel.transaction.Transactional
     [exec] 				+   property   propagation='SUPPORTS'
     [exec] 				+   property   readOnly=true
     [exec] 			+   return     com.liferay.portal.kernel.search.BaseModelSearchResult
     [exec] 		~   method     updateTag(long,long,java.lang.String,com.liferay.portal.kernel.service.ServiceContext)
     [exec] 			+   annotated  com.liferay.portal.kernel.search.Indexable
     [exec] 				+   property   type='REINDEX'
     [exec] 	&lt;   interface  com.liferay.asset.kernel.service.AssetVocabularyLocalService
     [exec] 		+   method     fetchAssetVocabularyByReferenceCode(long,java.lang.String)
     [exec] 			+   access     abstract
     [exec] 			+   annotated  com.liferay.portal.kernel.transaction.Transactional
     [exec] 				+   property   propagation='SUPPORTS'
     [exec] 				+   property   readOnly=true
     [exec] 			+   return     com.liferay.asset.kernel.model.AssetVocabulary
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal-7.1.x/portal-kernel/portal-kernel.jar against /opt/dev/projects/github/liferay-portal-7.1.x/.gradle/caches/modules-2/files-2.1/com.liferay.portal/com.liferay.portal.kernel/2.111.0/a0b90a176442724fa13b016240daac07911a3dc5/com.liferay.portal.kernel-2.111.0.jar</code></pre></p></div></div></li></ol></div></li><li><div><h5><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/600/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-integration-mysql57-jdk8</a></h5><div><h6>Job Results:</h6><p>1766 Tests Passed.<br/>89 Tests Failed.</p></div><ol><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/600//consoleText">AXIS_VARIABLE=0,label_exp=!master #600</a><ol><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/600//testReport/com.liferay.commerce.account.service.test/CommerceAccountLocalServiceTest/testAddBusinessCommerceAccount">CommerceAccountLocalServiceTest.testAddBusinessCommerceAccount</a><pre><code>java.lang.AssertionError: {level=ERROR, loggerName=com.liferay.commerce.product.internal.model.listener.CompanyModelListener, message=com.liferay.portal.kernel.exception.NoSuchUserException: No User exists with the key {companyId=50805, defaultUser=true}
	at com.liferay.portal.test.rule.LogAssertionAppender.append(LogAssertionAppender.java:61)
	at org.apache.log4j.AppenderSkeleton.doAppend(AppenderSkeleton.java:251)
	at org.apache.log4j.helpers.AppenderAttachableImpl.appendLoopOnAppenders(AppenderAttachableImpl.java:66)
	at org.apache.log4j.Category.callAppenders(Category.java:206)
	at org.apache.log4j.Category.forcedLog(Category.java:391)
	at org.apache.log4j.Category.log(Category.java:856)
	at com.liferay.portal.log.Log4jLogImpl.error(Log4jLogImpl.java:54)
	at com.liferay.portal.kernel.log.LogWrapper.error(LogWrapper.java:69)
	at com.liferay.commerce.product.internal.model.listener.CompanyModelListener.onAfterCreate(CompanyModelListener.java:55)
	at com.liferay.commerce.product.internal.model.listener.CompanyModelListener.onAfterCreate(CompanyModelListener.java:39)
	at com.liferay.portal.kernel.service.persistence.impl.BasePersistenceImpl.update(BasePersistenceImpl.java:360)
	at com.liferay.portal.service.impl.CompanyLocalServiceImpl.checkCompany(CompanyLocalServiceImpl.java:246)
	at com.liferay.portal.service.impl.CompanyLocalServiceImpl.addCompany(CompanyLocalServiceImpl.java:170)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMet...</code></pre></div></li><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/600//testReport/com.liferay.commerce.account.service.test/CommerceAccountLocalServiceTest/testAddPersonalCommerceAccount">CommerceAccountLocalServiceTest.testAddPersonalCommerceAccount</a><pre><code>java.lang.AssertionError: {level=ERROR, loggerName=com.liferay.commerce.product.internal.model.listener.CompanyModelListener, message=com.liferay.portal.kernel.exception.NoSuchUserException: No User exists with the key {companyId=50807, defaultUser=true}
	at com.liferay.portal.test.rule.LogAssertionAppender.append(LogAssertionAppender.java:61)
	at org.apache.log4j.AppenderSkeleton.doAppend(AppenderSkeleton.java:251)
	at org.apache.log4j.helpers.AppenderAttachableImpl.appendLoopOnAppenders(AppenderAttachableImpl.java:66)
	at org.apache.log4j.Category.callAppenders(Category.java:206)
	at org.apache.log4j.Category.forcedLog(Category.java:391)
	at org.apache.log4j.Category.log(Category.java:856)
	at com.liferay.portal.log.Log4jLogImpl.error(Log4jLogImpl.java:54)
	at com.liferay.portal.kernel.log.LogWrapper.error(LogWrapper.java:69)
	at com.liferay.commerce.product.internal.model.listener.CompanyModelListener.onAfterCreate(CompanyModelListener.java:55)
	at com.liferay.commerce.product.internal.model.listener.CompanyModelListener.onAfterCreate(CompanyModelListener.java:39)
	at com.liferay.portal.kernel.service.persistence.impl.BasePersistenceImpl.update(BasePersistenceImpl.java:360)
	at com.liferay.portal.service.impl.CompanyLocalServiceImpl.checkCompany(CompanyLocalServiceImpl.java:246)
	at com.liferay.portal.service.impl.CompanyLocalServiceImpl.addCompany(CompanyLocalServiceImpl.java:170)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMet...</code></pre></div></li><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/600//testReport/com.liferay.commerce.account.service.test/CommerceAccountLocalServiceTest/testBusinessOrganizationCommerceAccountsVisibility">CommerceAccountLocalServiceTest.testBusinessOrganizationCommerceAccountsVisibility</a><pre><code>java.lang.AssertionError: {level=ERROR, loggerName=com.liferay.commerce.product.internal.model.listener.CompanyModelListener, message=com.liferay.portal.kernel.exception.NoSuchUserException: No User exists with the key {companyId=50809, defaultUser=true}
	at com.liferay.portal.test.rule.LogAssertionAppender.append(LogAssertionAppender.java:61)
	at org.apache.log4j.AppenderSkeleton.doAppend(AppenderSkeleton.java:251)
	at org.apache.log4j.helpers.AppenderAttachableImpl.appendLoopOnAppenders(AppenderAttachableImpl.java:66)
	at org.apache.log4j.Category.callAppenders(Category.java:206)
	at org.apache.log4j.Category.forcedLog(Category.java:391)
	at org.apache.log4j.Category.log(Category.java:856)
	at com.liferay.portal.log.Log4jLogImpl.error(Log4jLogImpl.java:54)
	at com.liferay.portal.kernel.log.LogWrapper.error(LogWrapper.java:69)
	at com.liferay.commerce.product.internal.model.listener.CompanyModelListener.onAfterCreate(CompanyModelListener.java:55)
	at com.liferay.commerce.product.internal.model.listener.CompanyModelListener.onAfterCreate(CompanyModelListener.java:39)
	at com.liferay.portal.kernel.service.persistence.impl.BasePersistenceImpl.update(BasePersistenceImpl.java:360)
	at com.liferay.portal.service.impl.CompanyLocalServiceImpl.checkCompany(CompanyLocalServiceImpl.java:246)
	at com.liferay.portal.service.impl.CompanyLocalServiceImpl.addCompany(CompanyLocalServiceImpl.java:170)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMet...</code></pre></div></li><li>...</li></ol></div></li><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=1,label_exp=!master/600//consoleText">AXIS_VARIABLE=1,label_exp=!master #600</a><ol><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=1,label_exp=!master/600//testReport/com.liferay.commerce.price.list.internal.search.test/CommercePriceListIndexerTest/testSearch">CommercePriceListIndexerTest.testSearch</a><pre><code>java.lang.AssertionError: 
[
SCENARIO Test price list search
GIVEN I add a price list
WHEN I search for price lists
THEN The result will be 'only one', the price list added
]
[
Message: {level=ERROR, loggerName=com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper, message=Failed session factory verification
]
	at com.liferay.portal.test.rule.LogAssertionAppender.append(LogAssertionAppender.java:61)
	at org.apache.log4j.AppenderSkeleton.doAppend(AppenderSkeleton.java:251)
	at org.apache.log4j.helpers.AppenderAttachableImpl.appendLoopOnAppenders(AppenderAttachableImpl.java:66)
	at org.apache.log4j.Category.callAppenders(Category.java:206)
	at org.apache.log4j.Category.forcedLog(Category.java:391)
	at org.apache.log4j.Category.log(Category.java:856)
	at com.liferay.portal.log.Log4jLogImpl.error(Log4jLogImpl.java:54)
	at com.liferay.portal.kernel.log.LogWrapper.error(LogWrapper.java:69)
	at com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper._logFailure(VerifySessionFactoryWrapper.java:121)
	at com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper._verify(VerifySessionFactoryWrapper.java:181)
	at com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper.openSession(VerifySessionFactoryWrapper.java:98)
	at com.liferay.portal.kernel.service.persistence.impl.BasePersistenceImpl.openSession(BasePersistenceImpl.java:257)
	at com.liferay.portal.service.persistence.impl.GroupPersistenceImpl.fetchByC_C_C(GroupPersistenceImpl.java:6483)
	at co...</code></pre></div></li><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=1,label_exp=!master/600//testReport/com.liferay.commerce.price.list.service.test/CommercePriceEntryLocalServiceTest/testAddCommercePriceEntry1">CommercePriceEntryLocalServiceTest.testAddCommercePriceEntry1</a><pre><code>org.hibernate.exception.ConstraintViolationException: Could not execute JDBC batch update
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:96)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:275)
	at org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:268)
	at org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:184)
	at org.hibernate.event.def.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:321)
	at com.liferay.portal.dao.orm.hibernate.event.NestableFlushEventListener.onFlush(NestableFlushEventListener.java:61)
	at org.hibernate.impl.SessionImpl.flush(SessionImpl.java:1216)
	at com.liferay.portal.spring.hibernate.PortletTransactionManager$TransactionStatusWrapper.reset(PortletTransactionManager.java:260)
	at com.liferay.portal.spring.hibernate.PortletTransactionManager.commit(PortletTransactionManager.java:63)
	at com.liferay.portal.spring.transaction.DefaultTransactionExecutor._commit(DefaultTransactionExecutor.java:164)
	at com.liferay.portal.spring.transaction.DefaultTransactionExecutor._execute(DefaultTransactionExecutor.java:211)
	at com.liferay.portal.spring.transaction.DefaultTransactionExecutor.execute(DefaultTransactionExecutor.java:94)
	at com.liferay.portal.spring.transaction.TransactionInterceptor.invoke(TransactionInterceptor.java:113)
	at com.liferay.portal.spring...</code></pre></div></li><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=1,label_exp=!master/600//testReport/com.liferay.commerce.price.list.service.test/CommercePriceEntryLocalServiceTest/testAddCommercePriceEntry2">CommercePriceEntryLocalServiceTest.testAddCommercePriceEntry2</a><pre><code>org.hibernate.exception.ConstraintViolationException: Could not execute JDBC batch update
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:96)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:275)
	at org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:268)
	at org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:184)
	at org.hibernate.event.def.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:321)
	at com.liferay.portal.dao.orm.hibernate.event.NestableFlushEventListener.onFlush(NestableFlushEventListener.java:61)
	at org.hibernate.impl.SessionImpl.flush(SessionImpl.java:1216)
	at com.liferay.portal.spring.hibernate.PortletTransactionManager$TransactionStatusWrapper.reset(PortletTransactionManager.java:260)
	at com.liferay.portal.spring.hibernate.PortletTransactionManager.commit(PortletTransactionManager.java:63)
	at com.liferay.portal.spring.transaction.DefaultTransactionExecutor._commit(DefaultTransactionExecutor.java:164)
	at com.liferay.portal.spring.transaction.DefaultTransactionExecutor._execute(DefaultTransactionExecutor.java:211)
	at com.liferay.portal.spring.transaction.DefaultTransactionExecutor.execute(DefaultTransactionExecutor.java:94)
	at com.liferay.portal.spring.transaction.TransactionInterceptor.invoke(TransactionInterceptor.java:113)
	at com.liferay.portal.spring...</code></pre></div></li><li>...</li></ol></div></li><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=2,label_exp=!master/600//consoleText">AXIS_VARIABLE=2,label_exp=!master #600</a><ol><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=2,label_exp=!master/600//testReport/com.liferay.commerce.product.util.test/CPDefinitionHelperTest/testSearchCategory">CPDefinitionHelperTest.testSearchCategory</a><pre><code>java.lang.AssertionError: 
[
SCENARIO Search for CPDefinition by Category
GIVEN A collection of CPDefinitions
 AND A category
 AND Some of the CPDefinitions are associated to that category
WHEN I search for CPDefinitions given the category as a search filter
THEN The results will contain only those product linked to the category
]
[
Message: {level=ERROR, loggerName=com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper, message=Failed session factory verification
]
	at com.liferay.portal.test.rule.LogAssertionAppender.append(LogAssertionAppender.java:61)
	at org.apache.log4j.AppenderSkeleton.doAppend(AppenderSkeleton.java:251)
	at org.apache.log4j.helpers.AppenderAttachableImpl.appendLoopOnAppenders(AppenderAttachableImpl.java:66)
	at org.apache.log4j.Category.callAppenders(Category.java:206)
	at org.apache.log4j.Category.forcedLog(Category.java:391)
	at org.apache.log4j.Category.log(Category.java:856)
	at com.liferay.portal.log.Log4jLogImpl.error(Log4jLogImpl.java:54)
	at com.liferay.portal.kernel.log.LogWrapper.error(LogWrapper.java:69)
	at com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper._logFailure(VerifySessionFactoryWrapper.java:121)
	at com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper._verify(VerifySessionFactoryWrapper.java:181)
	at com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper.openSession(VerifySessionFactoryWrapper.java:98)
	at com.liferay.portal.kernel.service.persistence.impl.BasePersistenceImpl.open...</code></pre></div></li><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=2,label_exp=!master/600//testReport/com.liferay.commerce.product.util.test/CPDefinitionHelperTest/testSearchName">CPDefinitionHelperTest.testSearchName</a><pre><code>java.lang.AssertionError: 
[
SCENARIO Search for CPDefinitions by name
GIVEN A collection of CPDefinitions
WHEN I search for CPDefinitions given the name as a search filter
THEN The results will contain only those product with the name equal to the one in the search filter. In this specific case 1
]
[
Message: {level=ERROR, loggerName=com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper, message=Failed session factory verification
]
	at com.liferay.portal.test.rule.LogAssertionAppender.append(LogAssertionAppender.java:61)
	at org.apache.log4j.AppenderSkeleton.doAppend(AppenderSkeleton.java:251)
	at org.apache.log4j.helpers.AppenderAttachableImpl.appendLoopOnAppenders(AppenderAttachableImpl.java:66)
	at org.apache.log4j.Category.callAppenders(Category.java:206)
	at org.apache.log4j.Category.forcedLog(Category.java:391)
	at org.apache.log4j.Category.log(Category.java:856)
	at com.liferay.portal.log.Log4jLogImpl.error(Log4jLogImpl.java:54)
	at com.liferay.portal.kernel.log.LogWrapper.error(LogWrapper.java:69)
	at com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper._logFailure(VerifySessionFactoryWrapper.java:121)
	at com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper._verify(VerifySessionFactoryWrapper.java:181)
	at com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper.openSession(VerifySessionFactoryWrapper.java:98)
	at com.liferay.portal.kernel.service.persistence.impl.BasePersistenceImpl.openSession(BasePersistenceImpl.java...</code></pre></div></li><li><div><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=2,label_exp=!master/600//testReport/com.liferay.commerce.product.util.test/CPDefinitionHelperTest/testSearchStar">CPDefinitionHelperTest.testSearchStar</a><pre><code>java.lang.AssertionError: 
[
SCENARIO Search for CPDefinitions without filters
GIVEN A collection of CPDefinitions
WHEN I search for CPDefinitions without filters
THEN The results will contain all the products available
]
[
Message: {level=ERROR, loggerName=com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper, message=Failed session factory verification
]
	at com.liferay.portal.test.rule.LogAssertionAppender.append(LogAssertionAppender.java:61)
	at org.apache.log4j.AppenderSkeleton.doAppend(AppenderSkeleton.java:251)
	at org.apache.log4j.helpers.AppenderAttachableImpl.appendLoopOnAppenders(AppenderAttachableImpl.java:66)
	at org.apache.log4j.Category.callAppenders(Category.java:206)
	at org.apache.log4j.Category.forcedLog(Category.java:391)
	at org.apache.log4j.Category.log(Category.java:856)
	at com.liferay.portal.log.Log4jLogImpl.error(Log4jLogImpl.java:54)
	at com.liferay.portal.kernel.log.LogWrapper.error(LogWrapper.java:69)
	at com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper._logFailure(VerifySessionFactoryWrapper.java:121)
	at com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper._verify(VerifySessionFactoryWrapper.java:181)
	at com.liferay.portal.dao.orm.hibernate.VerifySessionFactoryWrapper.openSession(VerifySessionFactoryWrapper.java:98)
	at com.liferay.portal.kernel.service.persistence.impl.BasePersistenceImpl.openSession(BasePersistenceImpl.java:257)
	at com.liferay.portal.service.persistence.impl.GroupPersistenceImpl.fetc...</code></pre></div></li><li>...</li></ol></div></li></ol></div></li></ol></details></html>

--- a/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_definition_account_groups.jsp
+++ b/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_definition_account_groups.jsp
@@ -37,6 +37,7 @@ long[] commerceAccountGroupIds = cpDefinitionAccountGroupDisplayContext.getComme
 			<aui:input name="<%= Constants.CMD %>" type="hidden" value="updateAccountGroups" />
 			<aui:input name="redirect" type="hidden" value="<%= currentURL %>" />
 			<aui:input name="cpDefinitionId" type="hidden" value="<%= cpDefinitionId %>" />
+			<aui:input name="commerceAccountGroupIds" type="hidden" />
 
 			<div class="lfr-form-content">
 				<aui:fieldset-group markupView="lexicon">
@@ -46,7 +47,7 @@ long[] commerceAccountGroupIds = cpDefinitionAccountGroupDisplayContext.getComme
 						for (CommerceAccountGroup commerceAccountGroup : commerceAccountGroups) {
 						%>
 
-							<aui:input checked="<%= ArrayUtil.contains(commerceAccountGroupIds, commerceAccountGroup.getCommerceAccountGroupId()) %>" label="<%= commerceAccountGroup.getName() %>" name="commerceAccountGroupIds" type="checkbox" value="<%= commerceAccountGroup.getCommerceAccountGroupId() %>" />
+							<aui:input checked="<%= ArrayUtil.contains(commerceAccountGroupIds, commerceAccountGroup.getCommerceAccountGroupId()) %>" label="<%= commerceAccountGroup.getName() %>" name='<%= "commerceAccountGroupId" + commerceAccountGroup.getCommerceAccountGroupId() %>' onChange='<%= renderResponse.getNamespace() + "fulfillCommerceAccountGroupIds();" %>' type="checkbox" value="<%= commerceAccountGroup.getCommerceAccountGroupId() %>" />
 
 						<%
 						}
@@ -64,3 +65,14 @@ long[] commerceAccountGroupIds = cpDefinitionAccountGroupDisplayContext.getComme
 		</aui:form>
 	</c:otherwise>
 </c:choose>
+
+<aui:script>
+	function <portlet:namespace />fulfillCommerceAccountGroupIds(e) {
+		var form = AUI.$(document.<portlet:namespace />fm);
+		var values = Liferay.Util.listCheckedExcept(form, '<portlet:namespace />allRowIds');
+		form.fm('commerceAccountGroupIds').val(values);
+		return values;
+	}
+
+	<portlet:namespace />fulfillCommerceChannelIds();
+</aui:script>

--- a/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_definition_channels.jsp
+++ b/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_definition_channels.jsp
@@ -37,6 +37,7 @@ long[] commerceChannelIds = cpDefinitionChannelDisplayContext.getCommerceChannel
 			<aui:input name="<%= Constants.CMD %>" type="hidden" value="updateChannels" />
 			<aui:input name="redirect" type="hidden" value="<%= currentURL %>" />
 			<aui:input name="cpDefinitionId" type="hidden" value="<%= cpDefinitionId %>" />
+			<aui:input name="commerceChannelIds" type="hidden" />
 
 			<div class="lfr-form-content">
 				<aui:fieldset-group markupView="lexicon">
@@ -46,7 +47,7 @@ long[] commerceChannelIds = cpDefinitionChannelDisplayContext.getCommerceChannel
 						for (CommerceChannel commerceChannel : commerceChannels) {
 						%>
 
-							<aui:input checked="<%= ArrayUtil.contains(commerceChannelIds, commerceChannel.getCommerceChannelId()) %>" label="<%= commerceChannel.getName() %>" name="commerceChannelIds" type="checkbox" value="<%= commerceChannel.getCommerceChannelId() %>" />
+							<aui:input checked="<%= ArrayUtil.contains(commerceChannelIds, commerceChannel.getCommerceChannelId()) %>" label="<%= commerceChannel.getName() %>" name='<%= "commerceChannelId_" + commerceChannel.getCommerceChannelId() %>' onChange='<%= renderResponse.getNamespace() + "fulfillCommerceChannelIds();" %>' type="checkbox" value="<%= commerceChannel.getCommerceChannelId() %>" />
 
 						<%
 						}
@@ -64,3 +65,14 @@ long[] commerceChannelIds = cpDefinitionChannelDisplayContext.getCommerceChannel
 		</aui:form>
 	</c:otherwise>
 </c:choose>
+
+<aui:script>
+	function <portlet:namespace />fulfillCommerceChannelIds(e) {
+		var form = AUI.$(document.<portlet:namespace />fm);
+		var values = Liferay.Util.listCheckedExcept(form, '<portlet:namespace />allRowIds');
+		form.fm('commerceChannelIds').val(values);
+		return values;
+	}
+
+	<portlet:namespace />fulfillCommerceChannelIds();
+</aui:script>

